src/little_dict.jl

```diff
@@ -117,6 +117,15 @@ function Base.getkey(dd::LittleDict, key, default)
     end
 end
 
+function Base.map!(f, iter::Base.ValueIterator{<:LittleDict})
+    dict = iter.dict
+    vals = dict.vals
+    for i = 1:length(vals)
```

Comment:
```suggestion
    for i in 1:length(vals)
```

for loops with equality are a akward and ugly matlabism